src/common/models/recurringGift.model.spec.js

```diff
@@ -381,6 +381,10 @@ describe('recurringGift model', () => {
     it('should return the old draw date when start month and transaction day are unchanged', () => {
       expect(giftModel.nextGiftDate.toString()).toEqual(moment('2015-05-06').toString());
     });
+    it('should handle the case when the parent donation doesn\'t have a next draw date', () => {
+      delete giftModel.parentDonation['next-draw-date'];
+      expect(giftModel.nextGiftDate).toBeUndefined();
```

Comment:
We could find a better value. If an error is thrown here, the view never finishes compiling and `{{ gift.nextGiftDate.format('ll') }}` gets spit out on the screen. When it is undefined I think the angular template handles it gracefully and it leaves a blank here
![screen shot 2017-02-07 at 12 19 53 pm](https://cloud.githubusercontent.com/assets/756501/22709746/c411cc94-ed2f-11e6-8692-105d03313201.png)
I've only tested this on the payment-methods.html page. If moment functions are called in the template we should be fine. If we do something in js, we're going to see an error.

Hopefully the errored gifts get removed from the views and this won't be hit much.